src/plugins/intel_gpu/src/kernel_selector/kernels/lstm/lstm_seq_kernel_bfyx.cpp

```diff
@@ -0,0 +1,33 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "lstm_seq_kernel_bfyx.h"
+#include "kernel_selector_utils.h"
+
+namespace kernel_selector {
+
+ParamsKey LSTMSeqKernel_bfyx::GetSupportedKey() const {
```

Comment:
It seems like having separate LSTMSeqKernel* and LSTMCellKernel* is not really needed. You can probably have single kernel and kernel selector, and just add this `sequential` knob as a part of `lstm_params` structure and just set it from lstm_cell/seq ocl primitive_impl

Reply:
done